docs/sources/metainfo/component.xml

```diff
@@ -143,6 +143,32 @@
 		</listitem>
 		</varlistentry>
 
+		<varlistentry id="tag-icon">
+		<term>&lt;icon/&gt;</term>
+		<listitem>
+			<para>
+				The <code>&lt;icon/&gt;</code> tag describes the component icon. It is mostly used for GUI applications (component-type <literal>desktop-application</literal>).
+				It can be of the type <literal>stock</literal>, <literal>local</literal>,
+				or <literal>url</literal>.
+			</para>
+			<para>
+				<literal>stock</literal> icons are loaded from stock. The icon name should never include any file-extension or path.
+			</para>
+			<para>
+				<literal>local</literal> icons are loaded from a file in the filesystem.
+				They should specify a full file path in either <code>/usr/share/pixmaps</code>
+                                or <code>/usr/share/icons</code>. (These restrictions on the icon location are in
```

Comment:
Those restrictions are appstream-builder specific, I guess - the `local` type explicitly makes no restrictions for where icons are located, it could be anywhere on the filesystem. The appstream-generator tool deals with them whereever they are.
For the paths you mention, `stock` is the better solution anyway.
In general, using `local` is discouraged, `stock` is almost always the better option.

Reply:
I have removed that sentence.